lib/rules/alpha-value-notation/index.js

```diff
@@ -120,6 +120,7 @@ const rule = (primary, secondaryOptions, context) => {
 
 				report({
 					message: messages.expected(unfixed, fixed),
```

Comment:
[suggest] Performance tip: we can avoid extra evaluations by passing a function instead of a string (as a result of a function call):

```suggestion
					message: messages.expected,
```

The same is true for the other rules.

Reply:
@ybiquitous Done!

(sidenote: for some reason, I cannot re-request the review from both of you via github UI — it keeps rotating them; first time seeing this behavior — I wonder if this is the intended behavior or a UI bug?)